logstash-core/lib/logstash/pipeline.rb

```diff
@@ -38,6 +38,7 @@ def self.validate_config(config_str, settings = {})
     begin
       # There should be a better way to test this
       self.new(config_str, settings)
+      true
     rescue => e
       e.message
```

Comment:
should we make this return false in case something is raised?
 
would it also be better to use a global rescue for this method?  smething like

```
def some_method(args)
  self.new(....)
rescue
 e.message
 false 
end
```

but i'm sure this has broader implications as expectations of methods using this validate function might be expecting this message somehow, 

what do you think?